test/unit/foreman/renderer/scope/report_test.rb

```diff
@@ -104,7 +104,7 @@ class ReportScopeTest < ActiveSupport::TestCase
           Bool: false
           Empty: ''
       OUT
-      assert_equal expected_yaml, @scope.report_render(format: :yaml)
+      assert_equal YAML.safe_load(expected_yaml), YAML.safe_load(@scope.report_render(format: :yaml))
```

Comment:
I'm leaning to using a data structure in the test instead of parsing twice. Thoughts?

Reply:
You mean "expected" should be stored as a Hash?

Reply:
Yes

Reply:
done